docs/CreateAKSClusterForScalarDL.md

```diff
@@ -17,36 +17,50 @@ When deploying ScalarDL Ledger, you must:
+
+{% capture notice--info %}
+**Note**
+
+If you place the worker nodes in different [availability zones](https://learn.microsoft.com/en-us/azure/availability-zones/az-overview) (AZs), you can withstand an AZ failure.
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
 ### Use 4vCPU / 8GB memory nodes for the worker node in the ScalarDL Ledger node pool
 
-From the perspective of commercial licenses, resources for one pod running ScalarDL Ledger are limited to 2vCPU / 4GB memory. In addition, we recommend deploying one ScalarDL Ledger pod and one Envoy pod on one worker node.
+From the perspective of commercial licenses, resources for one pod running ScalarDL Ledger are limited to 2vCPU / 4GB memory. In addition, some pods other than ScalarDL Ledger pods could exist on the worker nodes.
 
-In other words, the following components run on one worker node:
+In other words, the following components could run on one worker node:
 
 * ScalarDL Ledger pod (2vCPU / 4GB)
 * Envoy proxy
+* Monitoring components (if you deploy monitoring components such `kube-prometheus-stack`)
 * Kubernetes components
 
-With this in mind, you should use a worker node that has 4vCPU / 8GB memory resources. We recommend running only the above components on the worker node for ScalarDL Ledger. And remember, for Byzantine fault detection to work properly, you cannot deploy your application pods on the same AKS cluster as the ScalarDL Ledger deployment.
+With this in mind, you should use a worker node that has at least 4vCPU / 8GB memory resources and use at least three worker nodes for availability, as mentioned in [Create at least three worker nodes and three pods](#create-at-least-three-worker-nodes-and-three-pods).
 
-Note that you should configure resource limits based on your system's workload if the Envoy pod exceeds the above resource usage. In addition, you should consider scaling out the worker node and the ScalarDL Ledger pod if the ScalarDL Ledger pod exceeds the above resource usage and if latency is high (throughput is low) in your system.
+However, three nodes with at least 4vCPU / 8GB memory resources per node is the minimum for a production environment. You should also consider the resources of the AKS cluster (for example, the number of worker nodes, vCPUs per node, memories per node, ScalarDL Ledger pods, and pods for your application), which depend on your system's workload. In addition, if you plan to scale the pods automatically by using some features like [Horizontal Pod Autoscaling (HPA)](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/), you should consider the maximum number of pods on the worker node to decide on the worker node resources.
```

Comment:
Sorry, this is my mistake. 
In case of ScalarDL, users should not deploy `user application` on the same Kubernetes cluster as ScalarDL.

Also, this document is for ScalarDL Ledger (Ledger only mode). So, the correct description is as follows.

```
(for example, the number of worker nodes, vCPUs per node, memory per node, and ScalarDL Ledger pods)
```

I fixed it in 64a19d4032cf160c9fa2269fbee20e22da8cc27a.